lib/github_advisory_sync.rb

```diff
@@ -353,7 +354,7 @@ def create(package)
         file.write new_data.merge(
           "patched_versions" => vulnerabilities,
           "related" => {
-            "url"  => advisory["references"]
+            "url"  => advisory["references"].map! { |url| url['url'] }
```

Comment:
You probably want `.map` which returns a new `Array`, instead of modifying in-place which changes the contents of `advisory["references"]` permanently.